packages/components/date-picker/stories/date-picker.happo.js

```diff
@@ -0,0 +1,189 @@
+import { html } from 'lit-html';
+import '@tradeshift/elements';
+import '@tradeshift/elements.date-picker';
+import '@tradeshift/elements/src/vars.css';
+
+export default {
+	title: 'ts-date-picker'
+};
+
+export const TestDatePickerSuite1 = () => {
+	return html`
+		<style>
+			.render-block {
+				flex: 0 0 19%;
+				height: 400px;
+			}
+			.render-block > div:first-child {
+				margin-top: 400px;
+			}
+		</style>
+		<div style="display: flex; flex-flow: row wrap; justify-content: space-between;">
+			<div class="render-block">
+				<p>Date picker</p>
+				<div style="margin-bottom: 400px">
+					Closed:
+					<ts-date-picker></ts-date-picker>
+				</div>
+
+				<div>
+					Opened: (deliberately testing showing current month with today dot, it needs manual approval every time it
+					runs in a different day)
+					<ts-date-picker opened></ts-date-picker>
```

Comment:
I'm not sure if it would be better to mock Date, to not have it fail? or is it okay to accept the change everytime?

Reply:
I personally prefer to mock dates, because tests that are always failing could distract when something else really fails. If you really want to test that it correctly renders on different dates, i think it is better to create several tests that have mocks with different dates :)
Because this date comes from outside of the component, I think it doesn't matter, was the date a true real one or we mocked it.